Package.swift

```diff
@@ -0,0 +1,31 @@
+// swift-tools-version: 5.9
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "mParticle-Apptentive",
+    platforms: [ .iOS(.v13) ],
```

Comment:
Why is deployment target iOS 13 for SPM but iOS 11 for Cocoapods?

Reply:
Good catch https://github.com/apptentive/apptentive-kit-ios/blob/main/ApptentiveKit.podspec